content/news/2022-07-29-bevy-0.8/index.md

```diff
@@ -0,0 +1,1131 @@
+
+This new representation sped up schedule construction by ~30 percent!
+
+**TODO: insert graph here**
+
+This change also removed a number of trait requirements from our label derives:
+
+```rust
+// Old (Bevy 0.7)
+#[derive(SystemLabel, Debug, Clone, Copy, PartialEq, Eq, Hash)]
+enum MovementSystem {
+    Velocity,
+    Gravity,
+}
+
+// New (Bevy 0.7)
+#[derive(SystemLabel, Clone)]
+enum MovementSystem {
+    Velocity,
+    Gravity,
+}
+```
+
+[`SystemLabel`]: https://docs.rs/bevy/0.8.0/bevy/ecs/schedule/trait.SystemLabel.html
+[`StageLabel`]: https://docs.rs/bevy/0.8.0/bevy/ecs/schedule/trait.StageLabel.html
+[`AppLabel`]: https://docs.rs/bevy/0.8.0/bevy/app/derive.AppLabel.html
+
+## Bevy Reflection Improvements
+
+Bevy's "Rust reflection" system: `bevy_reflect` is a core, foundational piece of Bevy's scene system. It provides a way to dynamically interact with Rust types at run-time without knowing their actual types. We've invested heavily in it this release in preparation for scripting support and scene system improvements.
+
+[`bevy_reflect`](https://crates.io/crates/bevy_reflect) aims to be a "generic" Rust reflection system. It can be used without Bevy. We believe it fills a very real gap in the Rust ecosystem and we encourage the wider Rust community to use it (and contribute!). 
+
+### "Untyped" Reflection
+
+<div class="release-feature-authors">authors: @jakobhellermann</div>
+
+The [`Reflect`] derives now automatically add a new [`ReflectFromPtr`] struct to the [`TypeRegistry`] for each reflected type. This enables using the new [untyped ECS apis](#scripting-modding-progress-untyped-ecs-apis) in combination with the reflection system. Thi helps enable things like 3rd party scripting and modding.
+
+[`Reflect`]: https://docs.rs/bevy/0.8.0/bevy/reflect/trait.Reflect.html
+[`ReflectFromPtr`]: https://docs.rs/bevy/0.8.0/bevy/reflect/struct.ReflectFromPtr.html
+[`TypeRegistry`]: https://docs.rs/bevy/0.8.0/bevy/reflect/struct.TypeRegistry.html
+
+### Default Trait Reflection
+
+<div class="release-feature-authors">authors: @jakobhellermann</div>
+
+It is now possible to construct [`Reflect`] types using their [`Default`] trait impl, provided they register it as part of the [`Reflect`] derive:
+
+```rust
+#[derive(Reflect, Default)]
+#[reflect(Default)]
+struct MyStruct {
+    foo: String,
+    bar: usize,
+}
+
+let registration = type_registry.get(TypeId::of::<MyStruct>()).unwrap();
+let reflect_default = registration.data::<ReflectDefault>().unwrap();
+// This contains a MyStruct instance with default values
+let my_struct: Box<dyn Reflect> = reflect_default.default();
+```
+
+This enables constructing components for entities without _any_ compile time information, which is useful for dynamic scenarios like scripting and scenes.
+
+[`Default`]: https://doc.rust-lang.org/std/default/trait.Default.html
+
+### Array Reflection
+
+<div class="release-feature-authors">authors: @MrGVSV</div>
```

Comment:
Just wanted to make it known that I merely rebased this PR. The original (https://github.com/bevyengine/bevy/pull/2383) was authored by @NathanSWard.

```suggestion
<div class="release-feature-authors">authors: @NathanSWard, @MrGVSV</div>
```

Reply:
Much appreciated!